### DISCUSSION TITLE
Removing quotes in translated strings

### DISCUSSION THREAD

**Discussion by raiguard** [crowdin/github-action#109] [Q&A] [answered]
Greetings.

Some of the languages that my project is getting translated into are getting [quotes added around their contents](https://github.com/raiguard/krastorio-2/blob/1fa12a7617470dcc7b9d9ad1bc8a0690b4f43077/locale/ja/Krastorio2.cfg#L562) that don't exist in the source files. I assume that this is because I'm using INI formatted files, which probably need that escaping. However, the game that consumes these files doesn't actually follow the INI standard, and quotes that are placed in the translations show up in game:

![image](https://user-images.githubusercontent.com/3515394/158001973-73e63870-8e8e-46bf-bd49-6af6d8e1b6c9.png)

(the quotation mark at the beginning should not be there)

I haven't found anything in the action's settings or Crowdin's settings to prevent this. Is there anything I can do?

Ref: https://github.com/raiguard/krastorio-2/issues/230

**Comment by Natalia936** [comment]
Hi @raiguard ! Could you please tell us the project name and which source file causes such an inconvenience?

**Reply by raiguard** [reply]
The project is [krastorio-2](https://crowdin.com/project/krastorio-2), and all of the source files are affected. My original post contains a link to the translation for the screenshot I posted in the description paragraph.

**Comment by Natalia936** [comment]
Quotes are added to all the strings or just specific ones? Could you please copy the URL to such strings in such case and send us?

**Reply by raiguard** [reply]
Quotes are added to any string that contains square brackets `[]` in the translation. Factorio uses square brackets for "rich text" allowing icons, fonts, colors, and other things to be modified.

[Here](https://crowdin.com/translate/krastorio-2/14/en-ja?filter=basic&value=5#q=water%20filtration) is the translation I showed in my screenshot, [here](https://github.com/raiguard/krastorio-2/blob/8c794c0e6852eb018c7ee09cd0fc556c1bfdd7e9/locale/en/Krastorio2.cfg#L562) is the source string, and [here](https://github.com/raiguard/krastorio-2/blob/1fa12a7617470dcc7b9d9ad1bc8a0690b4f43077/locale/ja/Krastorio2.cfg#L562) is the result in the translated file.

**Comment by Natalia936** [comment]
Thank you so much, will check it with devs 😉

**Comment by Natalia936** [comment]
Hi @raiguard , I've checked everything with our devs and they say that quotes are added by default to the strings that contain such symbols as #;=

Hope for your kind understanding on the matter

**Reply by raiguard** [reply]
So there is absolutely no way for the action to remove them? If not then I guess I'll have to whip up some sort of shell script to do it.

**Comment by Natalia936** [comment, marked as answer]
Hi @raiguard , sorry to say so but no, it's a default behavior of our system

**Reply by raiguard** [reply]
Ok, thanks for looking in to it. I'll think of something.

**Comment by Natalia936** [comment]
You're always welcome :)
